### PR TITLE
Katar: The unarmed skill weapon

### PR DESCRIPTION
**Pull request by john-pawson**
## About The Pull Request

It adds a new weapon, the Katar! a blade that rests above the fist, it has a thrust and cut attack, fast speed, low damage, and uses the unarmed skill. It can be crafted for 1 steel and smelted for 1 steel too.

kataringame
Photo of the katar sprite
kataringame2
Photo of how the katar looks while being held.
## Why It's Good For The Game

It adds more variety to the amount of choices for weapons in the game, adds a choice for unarmed skill weapons (stabbing and cutting with the katar is almost exactly the same movement and technique as throwing a punch), a skill that was rarely to almost never used.
It's also pretty fuckin cool weapon.

### CODE REVIEW

**Comment by Stagrawl** [comment]
+1

**Comment by Coolgat3** [comment]
a gravity knife for rogues would be cooler imo, we should honestly make gloves work with unarmed, we have plate gauntlets and chain gauntlets and i think none of them do anything about unarmed damage

File: code/game/objects/items/rogueweapons/melee/special.dm [hunk 1]
```diff
@@ -170,3 +170,64 @@
...
+	item_d_type = "slash"
+
+/datum/intent/katar/thrust
+	name = "thrust"
+	icon_state = "instab"
+	attack_verb = list("thrusts")
+	animname = "stab"
+	blade_class = BCLASS_STAB
+	hitsound = list('sound/combat/hits/bladed/genstab (1).ogg', 'sound/combat/hits/bladed/genstab (2).ogg', 'sound/combat/hits/bladed/genstab (3).ogg')
+	penfactor = 40
```
[the remark this answers is not in the document]
Comment (john-pawson) [reply code/game/objects/items/rogueweapons/melee/special.dm:221]: It's the exact same penfactor for the knives, wich i took as the weapon that is the most similar to the knives. Not only that but the katar is MAINLY a thrusting weapon, so it makes sense it has a high penfactor

**Comment by john-pawson** [comment]
> a gravity knife for rogues would be cooler imo, we should honestly make gloves work with unarmed, we have plate gauntlets and chain gauntlets and i think none of them do anything about unarmed damage

I don't see how gravity knives for rogues have anything to do with the katar or the unarmed skill. Or how the gloves should affect unarmed. But it would be a nice thing for another PR perhaps

**Pull request merged**